agent/acs/handler/acs_handler_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	"context"
 
 	"github.com/aws/amazon-ecs-agent/agent/api"
+	c "github.com/aws/amazon-ecs-agent/agent/api/container"
```

Comment:
How about doing something like `. "github.com/aws/amazon-ecs-agent/agent/api/container"`, so that we can reference the types directly here without using package name?

Reply:
we need to do similar refactoring for task and eni under api, so I don't want to give priority to container with . import reference.